src/Conf.ml

```diff
@@ -188,6 +188,13 @@ let quiet =
   let default = false in
   mk ~default Arg.(value & flag & info ["q"; "quiet"] ~doc)
 
+let dont_check_comments =
+  let doc =
+    "UNSAFE: Don't perform any check about comments & doc-comments"
```

Comment:
I don't know if using punctuation here messes up the man page generation, so maybe "UNSAFE: Do not perform any checking of comments and documentation comments."